pkg/utils/validation/cidr/disjoint.go

```diff
@@ -70,6 +76,43 @@ func ValidateNetworkDisjointedness(fldPath *field.Path, shootNodes, shootPods, s
 	return allErrs
 }
 
+// ValidateShootNetworkDisjointedness validates that the given shoot network is disjoint.
+func ValidateShootNetworkDisjointedness(fldPath *field.Path, shootNodes, shootPods, shootServices *string) field.ErrorList {
```

Comment:
Why is the `DefaultVpnRange` not checked in this function for overlaps?

Reply:
That is already being checked in ValidateNetworkDisjointedness() function so not needed here.

Reply:
Ok.